go/enclave/rollupchain/rollup_chain.go

```diff
@@ -571,11 +571,16 @@ func (rc *RollupChain) ExecuteOffChainTransaction(encryptedParams common.Encrypt
 		return nil, fmt.Errorf("required at least 1 params, but received %d", len(paramList))
 	}
 
-	callMsg, err := gethencoding.ExtractEthCall(paramList[0])
+	apiArgs, err := gethencoding.ExtractEthCall(paramList[0])
 	if err != nil {
 		return nil, fmt.Errorf("unable to decode EthCall Params - %w", err)
 	}
 
+	callMsg, err := apiArgs.ToMessage(0, gethcommon.Big0)
```

Comment:
can you make these 2 values as params in the rollup_chan, populated with 0 initially.
and also add a todo here that these 2 values need to be considered during the gas implementation

Reply:
Done 👍 

While doing this I saw that we're not reusing the `ethconfig.Config` I think we might reuse that structure in the future, might be useful to use a similar grouping of configs as geths. But maybe better done on a separate PR.